asf_core_data/pipeline/mcs/generate_mcs_data.py

```diff
@@ -209,9 +220,11 @@ def get_mcs_installations(epc_version="none", refresh=False):
         file_list = [
             ("/" + object.key) for object in bucket.objects.filter(Prefix=folder)
         ]  # bit of a hack
-        file_prefix = keyword_to_path_dict[epc_version]
+        file_prefix = keyword_to_path_dict[epc_version].split("{")[0]
         matches = [
-            filename for filename in file_list if filename.startswith(file_prefix)
+            filename
+            for filename in file_list
+            if (re.split("[0-9]", filename)[0] == file_prefix)
```

Comment:
what is the intention here? :)

Reply:
Don't know, that's Chris' code that I took over.

Reply:
# 51